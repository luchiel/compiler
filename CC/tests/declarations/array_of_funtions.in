float (*a(int b))[10] {}
float *b(int b)[10] {}
float *(c(int b)[10]) {}
float (*d)(int b)[10];