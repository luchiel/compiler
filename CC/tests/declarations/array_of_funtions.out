type <int>
type <float>
type <void>
function a
	returns pointer to array [
				{int 10}
			] of <float>
	arguments, locals
		variable b of type <int>
	{{}}
function b
	returns array [
			{int 10}
		] of pointer to <float>
	arguments, locals
		variable b of type <int>
	{{}}
function c
	returns array [
			{int 10}
		] of pointer to <float>
	arguments, locals
		variable b of type <int>
	{{}}
type <0>
	pointer to function 
			returns array [
					{int 10}
				] of <float>
			arguments, locals
				variable b of type <int>
variable d of type <0>
